src/components/Layout/Seo.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Helmet } from 'react-helmet-async'
+
+type SeoProps = {
+  title?: string
+  description?: string
+  type?: string
+  name?: string
+}
+
+export const Seo: React.FC<SeoProps> = ({
+  title,
+  description = 'ShapeShift DAO | Your Web3 & DeFi Portal',
+  type = 'website',
+  name = 'ShapeShift',
+}) => {
+  return (
+    <Helmet>
```

Comment:
include `<meta property="og:image" content="https://cloudflare-ipfs.com/ipfs/%REACT_APP_CID_THUMBNAIL_JPG%">` somewhere in here so we don't break link previews